docs/arcadia-users-group/20240730-arcadiathemer/20240730-arcadiathemer.Rmd

```diff
@@ -0,0 +1,143 @@
+
+We'll cover the following topics:
+
+1. Installing `arcadiathemeR`
+2. Using `arcadiathemeR` to layer onto a `ggplot2` plot
+3. Accessing and using colors, palettes, and gradients
+4. Saving plots
+
+## Prerequisites and Installation
+1. You need `R` installed, and `arcadiathemeR` requires at least R version >= 4.0 (the package was built and tested with version 4.3.1). Install `R` for Mac OSx [here]
+2. You can use `RStudio` or a Jupyter notebook for creating plots. Install `RStudio` for Mac OSx [here] or Jupyter in a conda environment [following these instructions]()
+3. To use the custom fonts you need to download the `TTF` formatted font files and place in the `Users/YOURUSERNAME/Library/Fonts/` directory. You can also double click on the fonts to install them using FontBook. Check out the Arcadia Science Brand Assets page in Notion to find these. This should only need to be performed once even if the package is updated over time. **You must download the TTF formatted files to be compatible with `arcadiathemeR`.** If you want to download both the `TTF` and `OTF` formatted files to use both `arcadiathemeR` and the `arcadia-pycolor` python package that's fine - there will just be duplicate selections for each font in Illustrator. 
+4. Install `arcadiathemeR` with the `remotes` package:
+
+    ```
+    # install.packages("remotes")
+    remotes::install_github("Arcadia-Science/arcadiathemeR")
+    ```
+5. Load the package to use in your scripts with `library(arcadiathemeR)`. When you first load the package it will print a message about whether the custom fonts have been accessed and loaded correctly.
+
+## Layering onto an existing `ggplot2` plot
+
+There are two main functions you will use to layer onto an existing `ggplot2` plot to create plots adhering to the Arcadia style guide and access palettes. These are the `theme_arcadia()` and `scale` functions. The particular `scale` function differs on whether you use `color` or `fill` to access colors. Here is how you use these functions without changing the default arguments:
+
+```{r, example_plot}
+library(ggplot2)
+library(arcadiathemeR)
+
+ggplot(data=mtcars, aes(x=hp, y=mpg, color=as.factor(cyl))) +
+  geom_point() +
+  theme_arcadia() +
+  scale_color_arcadia()
+```
+
+If you want to change the color palette used, you can access this with the `palette_name` argument in the `scale` function: 
+
+```{r, palette_change}
+ggplot(data=mtcars, aes(x=hp, y=mpg, color=as.factor(cyl))) +
+  geom_point() +
+  theme_arcadia() +
+  scale_color_arcadia(palette_name = "primary")
+```
+
+We have specific font specifications for whether the represented data is categorical or numerical. In the next plot where the x-axis is different categories, you can see where the font isn't specified correctly:
```

Comment:
The overlapping text labels on the x-axis in the figure don't look great... can the size be of the figure be adjusted so they don't overlap?

Reply:
I don't know how to fix that with this type of web layout, however it's an example of how you can fix that by moving the legend position